main.go

```diff
@@ -166,7 +166,6 @@ func main() {
 
 	if err = (&cassandrabackup.CassandraBackupReconciler{
 		Client:    mgr.GetClient(),
-		Log:       ctrl.Log.WithName("controllers").WithName("CassandraBackup"),
```

Comment:
But if you don't do that you won't know where the log comes from, right ? Why does it make any difference ?

Reply:
I was looking at for a warning message and it was missing in logs. 
But now I'm not sure, may be just restart of operator helped to show it.
Logs look in this way:
```
time="2022-09-20T11:48:04Z" level=info msg="Waiting Rack to be running before continuing, we break ReconcileRack after updated statefulset" cluster=tf-cassandra-config dc-rack=dc1-rack1
2022-09-20T11:48:06.081Z        INFO    controller_cassandracluster     Reconciling CassandraCluster    {"Request.Namespace": "tf", "Request.Name": "tf-cassandra-analytics"}
```

Ideally we should use one logger... May be I will make a pull request to fix this instead.